tasks/15_confirm-ending/confirm-ending.js

```diff
@@ -0,0 +1,7 @@
+export function confirmEnding(str, target) {
+  if (str.substr(-target.length) === target) {
```

Comment:
if it works - it could be returned in simpler way

Reply:
i assume that you're talking about .endsWith() method, am i right? if so, then the description of the task states, that we are prohibited from using it. Should I look for a solution other than .endsWith()?

Reply:
i mean `return str.substr(-target.length) === target`